setup.py

```diff
@@ -44,16 +44,32 @@ def _minimal_ext_cmd(cmd):
             GIT_REVISION = out.decode('utf-8').strip()
         except OSError:
             GIT_REVISION = "Unknown"
+        try:
+            cmd = ['git', 'config', '--get', 'remote.origin.url']
+            out = _minimal_ext_cmd(cmd).decode('utf-8').strip()
+            start_idx = 0
+            for prefix in [b'git@github.com:', b'git://github.com/',
+                           b'http://github.com/', b'https://github.com/',
+                           b'ssh://git@github.com/']:
+                if out.startswith(prefix):
```

Comment:
That's what I thought too, but see Travis build 911.3 line 802 (the first of the two I linked to in my plea for help). It ran with the decode statement and without the `b`s and still errored ("first arg must be byte or tuple of bytes, not str", i.e. the opposite error from build 912.3).